packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -84,8 +84,10 @@ export default class extends EntityModel {
 
     const properties: PropertyObject = {
       [SYSTEM_TYPES.propertyType.title.baseUri]: title,
-      [SYSTEM_TYPES.propertyType.summary.baseUri]: summary,
-      [SYSTEM_TYPES.propertyType.index.baseUri]: index,
+      ...(summary
+        ? { [SYSTEM_TYPES.propertyType.summary.baseUri]: summary }
+        : {}),
+      ...(index ? { [SYSTEM_TYPES.propertyType.index.baseUri]: index } : {}),
```

Comment:
**Blocking:** I think this will prevent the index from being set to `0` (because `0` is falsy). This could be addressed by something like:
```suggestion
      ...(index !== undefined ? { [SYSTEM_TYPES.propertyType.index.baseUri]: index } : {}),
```

Reply:
I've addressed this in a PR stacked two layers above this one, but we should probably avoid merging this in nonetheless.